Homework_3.py

```diff
@@ -0,0 +1,46 @@
+# import os
```

Comment:
Please try transformer logic. Will be easy to do calculations on the temp tables in it.